### PR TITLE
Use EOF to prevent failures from unexpected characters in GHA's

### PR DESCRIPTION
**Pull request by rmtolmach**
## Summary

Error seen sometimes in CI runs:
```
/home/runner/work/_temp/3711f819-88c3-4533-82f2-1842aa10d536.sh: line 345: unexpected EOF while looking for matching `"'
Error: Process completed with exit code 2.
```
[Example from here](https://github.com/department-of-veterans-affairs/vets-api/actions/runs/11349304416/job/31565085162?pr=18846#annotation:8:357)

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/93566

## Testing done

- Reproduced error locally by using various characters in approval comments, until it was determined that single quotes were causing the action to fail, which matches the case from above and [this one](https://github.com/department-of-veterans-affairs/vets-api/pull/18607#issuecomment-2377288194) from a few weeks ago, though further testing confirmed it was the single, not double quote in my comment. 

## What areas of the site does it impact?
Vets-api CI

### CODE REVIEW

**Comment by ryan-mcneil** [comment]
does "this" break it?

**Review by ryan-mcneil — APPROVED** [review]
does "this" break it?

**Review by ryan-mcneil — APPROVED** [review]

**Review by ryan-mcneil — APPROVED** [review]
Does this

👍

File: docs/setup/hybrid.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Developer Setup
 
-In hybrid mode, you will run vets-api natively, but run Postgres and Redis in Docker. By doing so you avoid any challenges of installing these two software packages and keeping them upgraded to the appropriate version.
+In hybrid mode, you'll run vets-api natively, but run Postgres and Redis in Docker. By doing so, you avoid the challenges of installing these two software packages and having to keep them upgraded to the appropriate version.
```
Comment (ryan-mcneil) [inline docs/setup/hybrid.md:3]: Idk, I kinda have a thing against single quotes now

Comment (rmtolmach) [reply docs/setup/hybrid.md:3]: 😆

**Review by ryan-mcneil — APPROVED** [review]
I can use single quotes and I'M SO HAPPY

**Comment by rmtolmach** [comment]
Nice work, Ryan!!

**Pull request merged**